tests/src/Traits/HttpRequestTrait.php

```diff
@@ -64,7 +64,7 @@ protected function query($query, $server = NULL, array $variables = [], array $e
    * @return \Symfony\Component\HttpFoundation\Response
    *   The http response object.
    */
-  protected function batchedQueries(array $queries, $server = NULL) {
+  protected function batchedQueries(array $queries, \Drupal\graphql\Entity\Server $server = NULL) {
```

Comment:
There are two problems with `$server` argument here. First we shouldn't add a full namespace (fully qualified domain name) for method arguments. And second, we should use interface instead of specific class. Let's use interface instead, because if somebody wants to extend default `Server` class and use it, they wouldn't be able to do so. Let's use here `ServerInterface`, and make user namespace is added to the top of the file:
```
use Drupal\graphql\Entity\ServerInterface;
```